### PR TITLE
admin pages on react

### PR DESCRIPTION
**Pull request by andrewjcasal**
Finish all admin pages on react

### CODE REVIEW

File: src/Components/KeyPerformanceIndicators/KeyPerformanceIndicators.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  }]
+};
+
+let chartOptions = {
+  responsive: true,
+  datasetStrokeWidth : 3,
+  pointDotStrokeWidth : 4,
+  tooltipFillColor: "rgba(0,0,0,0.8)",
+  tooltipFontStyle: "bold",
+  tooltipTemplate: "<%if (label){%><%=label + ' hod' %>: <%}%><%= value + '°C' %>",
```
Comment (mKleinCreative) [inline src/Components/KeyPerformanceIndicators/KeyPerformanceIndicators.js]: this is EJS syntax.. is that part of chart.js?

File: src/Pages/BusinessDetails.js [hunk 2]
```diff
@@ -7,45 +7,52 @@ import Dropdown from '../Components/UI/Dropdown/Dropdown';
...
+              </Row>
+              <Row>
+                <Col md={12}><input type="text" placeholder="YOURCLUB.cinchsports.com" /></Col>
+              </Row>
+              <Row>
+                <Col md={12}><input type="text" placeholder="Address" /></Col>
+              </Row>
+              <Row>
+                <Col md={4}><input type="text" placeholder="City" /></Col>
+                <Col md={4}><Dropdown title="State" options={["male","female"]}/></Col>
```
Comment (mKleinCreative) [inline src/Pages/BusinessDetails.js]: put states here instead of male female? nbd

File: src/Pages/Users.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    accessor: "empty", 
+    width: "90",
+    Cell: row => (<div style={{marginLeft: 24 + "px"}}><input type="checkbox" /></div>)
+  },{
+    Header: "Image",
+    accessor: "player",
+    width: "94",
+    Cell: row => (<div className="avatar-image"></div>)
+  },
+  {Header: "Player Name",accessor: "playerName", width: "126"},
```
Comment (mKleinCreative) [inline src/Pages/Users.js]: this area's so wonky with formatting o.0

File: src/Pages/UsersDetail.js [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+  {Header: "Player Name",accessor: "playerName", width: "126"},
+  {Header: "ID",       accessor: "id", width: "77"},
+  {Header: "Price",       accessor: "price", width: "99"},
+  {Header: "Quan.",       accessor: "quantity", width: "68"},
+  {Header: "Total Price",           accessor: "total", width: "107"},
+  {Header: "Customer",           accessor: "customer", width: "75"},
+  {Header: "",           accessor: "view", width: "65",
+    Cell: row => (<button className="view-button">VIEW</button>)}
+]
+
```
Comment (mKleinCreative) [inline src/Pages/UsersDetail.js]: same complaint as before, but nbd

File: src/containers/App/App.scss [hunk 5]
```diff
@@ -70,18 +70,24 @@ body {
...
   width: 100%;
   height: 50px;
   margin-bottom: 0;
 }
+.wrapper > .row {
+  margin-left: -8px;
+  margin-right: -8px;
+
+  > div[class*="col-md-"] {
+    padding-left: 8px;
```
Comment (mKleinCreative) [inline src/containers/App/App.scss]: nicee

**Comment by mKleinCreative** [comment]
mostly superficial changes. looks good to me though.

File: src/Components/KeyPerformanceIndicators/KeyPerformanceIndicators.js (hunk 1 above)
Comment (andrewjcasal) [reply src/Components/KeyPerformanceIndicators/KeyPerformanceIndicators.js]: Yes! This is part of the templating system.

Comment (andrewjcasal) [reply src/Components/KeyPerformanceIndicators/KeyPerformanceIndicators.js]: screen shot 2018-03-13 at 11 11 28 pm

File: src/Pages/BusinessDetails.js (hunk 2 above)
Comment (andrewjcasal) [reply src/Pages/BusinessDetails.js]: Let's leave this, as we're going to integrate all the dropdowns over the next couple days anyway!

File: src/Pages/Users.js (hunk 3 above)
Comment (andrewjcasal) [reply src/Pages/Users.js]: Will fix.

File: src/Pages/UsersDetail.js (hunk 4 above)
Comment (andrewjcasal) [reply src/Pages/UsersDetail.js]: will fix.

**Pull request merged**